README.md

```diff
@@ -86,107 +86,99 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
 ## 📋 Requirements
 
-- **🍎 Apple M1/M2/M3 Mac or 🐧 Linux system** (tested on Fedora).
-  We anticipate support for more operating systems in the future.
+- **🍎 Apple M1/M2/M3 Mac or 🐧 Linux system** (tested on Fedora). We anticipate support for more operating systems in the future.
+
+   📋  When installing the `ilab` CLI on macOS, you may have to run the `xcode-select --install` command, installing the required packages previously listed.
```

Comment:
Are the required packages previously listed? I think they're below? (I'm a little confused on that phrase, "the required packages previously listed.")

Reply:
Oh I see. It's moved up :) So you might want to rephrase, "the required packages listed below."

Reply:
Nice catch!